lib/routes/home/widgets/payments_list/widgets/payment_details_sheet/widgets/payment_details_sheet_expiry.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:auto_size_text/auto_size_text.dart';
+//import 'package:breez_translations/breez_translations_locales.dart';
+//import 'package:breez_translations/generated/breez_translations.dart';
+import 'package:flutter/material.dart';
+import 'package:l_breez/utils/date.dart';
+
+class PaymentDetailsSheetExpiry extends StatelessWidget {
+  final DateTime expiryDate;
+  final AutoSizeGroup? labelAutoSizeGroup;
+
+  const PaymentDetailsSheetExpiry({
+    required this.expiryDate,
+    super.key,
+    this.labelAutoSizeGroup,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    //final BreezTranslations texts = context.texts();
+    final ThemeData themeData = Theme.of(context);
+
+    return Row(
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.only(right: 8.0),
+          child: AutoSizeText(
+            // TODO(danielgranhao): translate
+            'Est. expiry:',
```

Comment:
Please consult with Roy on the text.

Reply:
Thanks, I did, and the outcome is we want to use the same as on Breez mobile. I switched to using translations in [3ce4ba7](https://github.com/breez/misty-breez/pull/311/commits/3ce4ba706b34178c60939b8bdc0fc84f43610704).